adviser/base/argo-workflows/advise.yaml

```diff
@@ -103,6 +103,14 @@ spec:
               secretKeySecret:
                 name: argo-artifact-repository-secrets
                 key: secretKey
+      initContainers:
+        - name: prescriptions
+          image: prescriptions
+          command:
+            - /bin/sh
```

Comment:
we could also `git clone` from a prescriptions repo tag, like https://kubernetes.io/docs/concepts/storage/volumes/#gitrepo This would require an online connection to init any adviser workflow -1
mounting it from a configmap is no good, as the CM has a size limit of 1Mi -1

so the pattern us used seems to be good. what about writing it up in an ADR?

Reply:
ADR can land in the core repo, I will merge this to have the functionality available in the next release.

Thanks for your feedback.